lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -430,7 +430,7 @@ sub appears_on
         }
     }
 
-    return %map;
+    return \%map;
```

Comment:
Oh, sorry, with all the other things we talked about I forgot this one :) Thanks!